#
# The test case never succeeds if it does not delete interim results.
# Patch has been sent to upstream:
#	https://github.com/proftpd/proftpd/pull/713
# The patch will die with future re-sync.
#
diff --git a/tests/api/misc.c b/tests/api/misc.c
index 926d9b3e3..199883488 100644
--- a/tests/api/misc.c
+++ b/tests/api/misc.c
@@ -303,6 +303,7 @@ START_TEST (dir_readlink_test) {
     res = dir_readlink(p, path, buf, bufsz, flags);
     fail_unless(res == 0, "Failed to handle empty symlink");
   }
+  (void) unlink(path);
 
   /* Not chrooted, absolute dst path */
   memset(buf, '\0', bufsz);
